Install.md

```diff
@@ -55,6 +81,19 @@ We are sad to see you go!  Please give us some feedback on how we could do bette
 3. The value retrieved in _step 1_ contained the path to the JAR.  You can go delete that JAR.
 4. Archive the `insecureStore.xml` file from the `${project.artifactId}` sub-folder under your HOME folder.
 
+### Homebrew uninstallation
+
+1. Retrieve the value of the `credential.helper` configuration: `git config --global --get credential.helper ${project.artifactId}`
+2. Launch the JAR in `uninstall` mode, which will update the "global" Git configuration file (the one in your home folder): `${project.artifactId} uninstall`
+3. Uninstall the ${project.artifactId} package with Homebrew: `brew uninstall ${project.artifactId}`
+4. Archive the `insecureStore.xml` file from the `${project.artifactId}` sub-folder under your HOME folder.
+
+### RPM uninstallation
+
+1. Retrieve the value of the `credential.helper` configuration: `git config --global --get credential.helper ${project.artifactId}`

```

Comment:
This step is not necessary with a package manager because you don't need to know the path to the JAR to uninstall.